### PR TITLE
B0.20 develop

### PR DESCRIPTION
**Pull request by shadeofblue**

### CODE REVIEW

**Review by maaktweluit — APPROVED** [review]
Did not look, was already reviewed before

**Review by Wiezzel — COMMENTED** [review]
Why is the whole `Golem.aip` marked as changed? Did you mess with the line endings?

**Comment by shadeofblue** [comment]
> Why is the whole `Golem.aip` marked as changed? Did you mess with the line endings?

@Wiezzel I believe you did, originally... https://github.com/golemfactory/golem/commit/c410543273d05c5c24768cb23f7cbcb32debdb74#diff-bff879c88299bdfd74bd1032cba6e7d7

anyway, fixed

**Comment by shadeofblue** [comment]
anyway, @Wiezzel brought back your beloved Windows-style endings ;p

**Review by Wiezzel — APPROVED** [review]

**Comment by Wiezzel** [comment]
@shadeofblue This is probably the only file when using `CRLF` makes sense. It's auto-generated by Advanced Installer and to keep `LF` you need to change it manually.

File: apps/rendering/task/framerenderingtask.py [hunk 1]
```diff
@@ -75,6 +75,23 @@ def calculate_subtasks_count(
...
 
 
+@rpc_utils.expose('comp.task.subtasks.count')
+def legacy_calculate_subtasks_count(
+        subtasks_count: int,
+        optimize_total: bool,
+        use_frames: bool,
+        frames: list) -> int:
+    """
+    TODO: remove this before 0.21
```
Comment (jiivan) [inline apps/rendering/task/framerenderingtask.py:85]: Shall we remove this in develop?

**Review by jiivan — APPROVED** [review]
heavyrain_nod

File: apps/rendering/task/framerenderingtask.py (hunk 1 above)
Comment (shadeofblue) [reply apps/rendering/task/framerenderingtask.py:85]: no idea

**Pull request merged**